src/app/modules/main/stickers/view.nim

```diff
@@ -105,7 +105,7 @@ QtObject:
     self.installedStickerPacksLoaded = flag
 
   proc getInstalledStickerPacksLoaded*(self: View): bool =
-    return self.installedStickerPacksloaded
+    return self.installedStickerPacksLoaded
```

Comment:
@jrainville @saledjenic FYI

My favorite NIM typo so far... This things happily compiled since forever

... and broke reloading the installed sticker packs when going offline/online